notebooks/en/pipeline_notus_instructions_preferences_legal.ipynb

```diff
@@ -0,0 +1,810 @@
+{
```

Comment:
<p>typo: "is used to creates" =&gt; "is used to create"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/56/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>